src/styles/calendar.scss

```
@@ -0,0 +1,50 @@
+.calendar {
+  padding: 10px;
+
+  &__container {
+    display: flex;
+    gap: $gap;
+    flex-wrap: wrap;
+    list-style: none;
+    padding-left: 0;
+    width: (7 * $square-size) + 6 * $gap;
+
+    @for $day from 1 through 31 {
+      & :nth-child(n + #{$day}):before {
+        content: "#{$day}";
+        font-size: $day-font-size;
+        font-family: $day-font;
+      }
+    }
+
+    @for $i from 29 through 31 {
+      &--month-length--#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $square-size;
+    height: $square-size;
+    background-color: $day-color;
+    border: 1px solid #000;
+    box-sizing: border-box;
+    transition: background-color, transform 0.5s;
```

Fix the syntax to change background-color smoothly.
Also, It would be great to create a variable for transition-duration
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```

---

fixed